include/metal/list/sort.hpp

```diff
@@ -29,6 +31,8 @@ namespace metal {
     ///     using result = metal::sort<l, lbd>;
     /// \endcode
     ///
+    /// \tip{`lbd` may be omitted, in which case it defaults to `metal::lambda<metal::less>`.}
```

Comment:
This change introduces the first _note_. Here is what `grep -r '\\tip' include/` returns after the change:
```
include/metal/value/value.hpp:    /// \tip{Use `metal::is_value` to trigger \SFINAE.}
include/metal/value/value.hpp:    /// \tip{Use `metal::value` to prevent undesired template pattern matching.}
include/metal/list/sort.hpp:    /// \tip{`lbd` may be omitted, in which case it defaults to `metal::lambda<metal::less>`.}
include/metal/number/if.hpp:    /// \tip{If `num{} != false`, `y` may be omitted.}
include/metal/lambda/bind.hpp:    /// \tip{Use metal::arg<n> as a placeholder for the n-th argument.}
```
The `stable`-sorting information IMHO belongs to another category than these _tips_. On the other hand, the default-argument information seems like a good fit, do you agree?

Reply:
Makes sense to me.